dags/landsat_scenes_sync/landsat_scenes_gap_report.py

```diff
@@ -288,10 +282,10 @@ def generate_buckets_diff(landsat: str, file_name: str, update_stac: bool = Fals
         )
 
         logging.info(f"Number of missing scenes: {len(missing_scenes)}")
-        logging.info(f"Wrote missing scenes to: s3://{LANDSAT_SYNC_BUCKET_NAME}/{key}")
+        logging.info(f"Wrote missing scenes to: {LANDSAT_SYNC_BUCKET_NAME}/{key}")
 
         if len(orphaned_scenes) > 0:
-            output_filename = f"{landsat}_{date_string}_orphaned.txt.gz"
+            output_filename = f"{landsat}_{datetime.today().isoformat()}_orphaned.txt.gz"
```

Comment:
Can we please change this back to using a simple date.

Reply:
I thought you had added that... sorry